### PR TITLE
docs(review): docs content review

### PR DESCRIPTION
**Pull request by bene2k1**
### Description

Docs content review 2024/02/08

### CODE REVIEW

File: containers/kubernetes/how-to/access-kubernetes-dashboard.mdx [hunk 1]
```diff
@@ -24,19 +24,8 @@ categories:
...
-    * Download of the kubeconfig file,
-    * The option to delete the cluster.
-
-If you click the **Dashboard** button, you are taken to your cluster's Kubernetes dashboard. Here, you can troubleshoot your containerized application, manage the cluster resources, get an overview of applications running on your cluster, and create or modify certain resources.
-
-If you click the **Pools** tab, you are able to add, edit or delete a pool on your cluster.
-
-If you click the **Nodes** tab, you are able to reboot or replace nodes on your cluster.
-
+3. Click **Dashboard**, you are taken to your cluster's Kubernetes dashboard.
```
Comment (jcirinosclwy) [inline containers/kubernetes/how-to/access-kubernetes-dashboard.mdx:27]:
```suggestion
3. Click **Dashboard**. You are taken to your cluster's Kubernetes dashboard.
```

File: dedibox-network/network/how-to/subscribe-to-jmrp.mdx [hunk 2]
```diff
@@ -22,12 +22,12 @@ categories:
...
 By subscribing to the JMRP (Junk Mail Reporting Program), you can register your IPs with Microsoft's mail delivery system. This registration enables you to take appropriate actions in case your IP is identified as a source of spam, preventing it from getting blacklisted.
 
-By default, the authorities use the IP owner's information (Scaleway/Online.net) to determine the association of a mail address with a specific IP. 
+By default, the authorities use the IP owner's information (Scaleway/Online.net) to determine the association of a mail address with a specific IP.
 However, if you have properly [configured the reverse DNS of your IP](/dedibox-network/ip-failover/how-to/configure-reverse-dns)), you have the option to select an email address within your domain.
 
 By following these steps, you can effectively manage your IP reputation and maintain a clean email delivery system.
 
-1. Open the JMRP website in your web browser using this link: [JMRP interface](https://postmaster.live.com/snds/JMRP.aspx).
+1. Open the JMRP website in your web browser using this link: [JMRP interface](https://sendersupport.olc.protection.outlook.com/snds/JMRP.aspx).
```
Comment (jcirinosclwy) [inline dedibox-network/network/how-to/subscribe-to-jmrp.mdx:30]:
```suggestion
1. Open the [JMRP website](https://sendersupport.olc.protection.outlook.com/snds/JMRP.aspx) in your web browser.
```

Comment (jcirinosclwy) [reply dedibox-network/network/how-to/subscribe-to-jmrp.mdx:30]: Both works, so this is just a suggestion. :)

File: managed-databases/document-database/how-to/enable-autobackup.mdx [hunk 3]
```diff
@@ -7,13 +7,13 @@ content:
...
 dates:
-  validation: 2023-08-01
+  validation: 2024-02-08
   posted: 2023-08-01
 categories:
   - managed-databases
 ---
 
-With the Autobackup feature, the creation and management of your Database Instance snapshots is automated according to your Autobackup schedule configuration.
+With the autobackup feature, the creation and management of your Database Instance snapshots is automated according to your autobackup schedule configuration.
```
Comment (jcirinosclwy) [inline managed-databases/document-database/how-to/enable-autobackup.mdx:16]:
```suggestion
With the autobackup feature, the creation and management of your Database Instance snapshots is automated according to your schedule configuration.
```

File: managed-databases/document-database/how-to/enable-autobackup.mdx [hunk 4]
```diff
@@ -23,16 +23,16 @@ With the Autobackup feature, the creation and management of your Database Instan
 </Message>
 
 <Message type="note">
-  The Autobackup feature is enabled by default upon the creation of a Database Instance. Simultaneously, a schedule policy is created. The default Autobackup frequency is once per day with a retention period of 7 days. You can change this in the Database Instance information page later.
+  The autobackup feature is enabled by default upon the creation of a Database Instance. Simultaneously, a schedule policy is created. The default autobackup frequency is once per day with a retention period of 7 days. You can change this in the Database Instance information page later.
```
Comment (jcirinosclwy) [inline managed-databases/document-database/how-to/enable-autobackup.mdx:26]:
```suggestion
  The autobackup feature is enabled by default upon the creation of a Database Instance. Simultaneously, a schedule policy is created. The default autobackup frequency is once per day, with a retention period of 7 days. You can change this in the Database Instance information page later.
```

File: managed-databases/document-database/how-to/enable-autobackup.mdx [hunk 5]
```diff
@@ -23,16 +23,16 @@ With the Autobackup feature, the creation and management of your Database Instan
...
 </Message>
 
 1. Click **Document Databases** under **Managed Databases** on the side menu. A list of your Database Instances displays.
 2. Click the Database Instance name or <Icon name="more" /> > **More info** to access the Database Instance information page.
 3. Click <Icon name="edit" /> in the **Snapshots** tab to configure the backup frequency and retention.
     <Message type="note">
-      Although Autobackup is enabled by default, you can disable the feature before you create the Instance, by unchecking the box in the creation wizard. You can also disable it after the Database Instance is ready.
+      Although autobackup is enabled by default, you can disable the feature before you create the Instance, by unchecking the box in the creation wizard. You can also disable it after the Database Instance is ready.
```
Comment (jcirinosclwy) [inline managed-databases/document-database/how-to/enable-autobackup.mdx:35]:
```suggestion
      Although autobackup is enabled by default, you can disable the feature before creating an Instance, by unchecking the box in the creation wizard. You can also disable it after the Database Instance is ready.
```

File: managed-databases/document-database/how-to/manage-read-replica.mdx [hunk 6]
```diff
@@ -23,7 +23,7 @@ categories:
 
 ## How to resync a Read Replica
 
-When you resync a Read Replica, first it is reset, and then its data is resynchronized from the primary node. Your Read Replica will be unavailable during the resync process. The duration of this process is proportional to your Database Instance size. The configured endpoints will not change.
+When you resync a Read Replica, first it is reset, and then its data is resynchronized from the primary node. Your Read Replica will be unavailable during the resynchronization process. The duration of this process is proportional to your Database Instance size. The configured endpoints will not change.
```
Comment (jcirinosclwy) [inline managed-databases/document-database/how-to/manage-read-replica.mdx:26]:
```suggestion
When you resync a Read Replica, first it is reset, and then its data is resynchronized from the primary node. Your Read Replica will be unavailable during resynchronization. The process duration is proportional to the size of your Database Instance. The configured endpoints will not change.
```

**Review by jcirinosclwy — APPROVED** [review]

**Review by nerda-codes — APPROVED** [review]

**Pull request merged**